packages/balancer/agents/autonomous_fund/aea-config.yaml

```diff
@@ -170,13 +170,13 @@ models:
       service_id: pool_manager
       service_registry_address: null
       setup:
-        safe_contract_address: ${SKILL_AUTONOMOUS_FUND_ABCI_MODELS_PARAMS_ARGS_SETUP_SAFE_CONTRACT_ADDRESS:list:["0x8001bdCf80F8Fb61CdcDA48419A30b430B385ca1"]}
+        safe_contract_address: ${list:["0x8001bdCf80F8Fb61CdcDA48419A30b430B385ca1"]}
```

Comment:
Should we not make this `0x00.00` as defaukt?

Reply:
that address is actually correct, because this an updated version of the service.

Reply:
This is at the agent level. There is no sense in specifying a concrete address imo

Reply:
I'll create an issue for it, as it might end up requiring to update tests.

Reply:
#70